### DISCUSSION TITLE
Localization / Custom CSL-Data

### DISCUSSION THREAD

**Discussion by MaPeras** [knowledgefutures/pubpub#1800] [Ideas]
Hi folks,

first of all - thank you for this amazing software and the philosophy behind it! This really points towards a better system of academic publishing. I am part of a small NGO called [OpenRewi](https://openrewi.org/en/). We are a network of early-career-scholars, producing books about law under a free license. 

We are considering using PubPub for an experimental project together with another NGO dedicated to free knowledge. It will be a legal commentary on an important German legal act, which is a weird but common form of legal scholarship in Germany. You can imagine it as a software documentation, but for legal code. 

The only caveats for now would be:
- localization/translation of the interface - as you have written here #1698 - this does not seem to be a priority, right? 
- integrating [custom CSL styles](https://github.com/citation-style-language/styles). Your "Cite"-button is just wonderful, but citing in the german legal academy is again - really weird. Woud it be possible to implement such a feature in theory?

I realize the specifity of these questions. If we aquire funds for this, do you think we could cooperate on implementing them into PubPub? For starters we will for sure become KFG members, if we start working with PubPub full time.

Greetings from Europe
Max

**Comment by gabestein** [comment]
Hi Max,

Thanks for considering PubPub, and for the kind words. My sincere apologies for the delay answering your questions.

On localization/translation: yes, that's still our situation, unfortunately. If you were to acquire funding, we would absolutely love to prioritize localization and translation, and I know you wouldn't be alone in communities who would be willing to help, and possible support, the effort.

On CSL styles: as long as there's an official stylefile for it, it's super easy for us to add the style you need, and we wouldn't charge for it.

Let me know if I can answer any other questions.

Gabe

**Reply by MaPeras** [reply]
Hi Gabe!

> On localization/translation: yes, that's still our situation, unfortunately. If you were to acquire funding, we would absolutely love to prioritize localization and translation, and I know you wouldn't be alone in communities who would be willing to help, and possible support, the effort.

Great - we are in contact with a similar initiative from Switzerland and will get back to you directly, if we find some funds.

> On CSL styles: as long as there's an official stylefile for it, it's super easy for us to add the style you need, and we wouldn't charge for it.

That's wonderful, thank you! We are currently drafting a common CSL-style for our organization. When we submit it to the repository, I will let you know.

Cheers
Max

**Comment by MaPeras** [comment]
Small follow-up question

> On CSL styles: as long as there's an official stylefile for it, it's super easy for us to add the style you need, and we wouldn't charge for it.

Do you fetch/pull the recent version of the style from the csl database? We are thinking about writing a prototype, that would not be perfect and of course I dont want to ask you every two weeks to update it.

**Reply by gabestein** [reply]
We don’t keep them synced currently, but I can talk to some folks about how hard it would be to sub module a repo where you were actively developing it. 

Even so, it’s pretty easy to update - every two weeks would be a bit much potentially, but I don’t think we’d have much trouble if you wanted to update it monthly or something as you develop it.

**Comment by MaPeras** [comment]
Dear Gabe,

Thanks again for your kind replies and offers. We have discussed it intensely. For now we will use Word/LibreOffice first, and import our texts to PubPub during our last book sprint (as your import-function is pretty nice, too!).

Main reasons are:
- no track-changes-mode, which is important for our open and intense peer-review process
- no live updates with a Zotero Database, that we plan to build collaboratively 

Eventually, I would love to move the whole process to PubPub, but we will get back to you, when we apply for KFG-membership and can contribute some resources for development.

Cheers
Max

**Reply by gabestein** [reply]
Great! Sounds like the right decision for what you need. Some day soon, we'll to have better support for track changes and Zotero (or Zotero-like & compatible tooling) to get more of your workflow on PubPub.
